src/site/_data/paths/reliable.json

```
@@ -43,6 +43,15 @@
         "offline-ux-design-guidelines",
         "extending-workbox"
       ]
+    },
+    {
+      "title": "Window to worker communication (and vice versa)",
```

```suggestion
      "title": "Window to worker communication",
```
Changing this because it's messing up Markdown links:

    […](/reliable/#window-to-worker-communication-(and-vice-versa))